app/views/templates/tests-by-week-widget.html.slim

```diff
@@ -0,0 +1,19 @@
+.panel.panel-default
+  .panel-heading
+    h3.panel-title Tests suite size
+  .panel-body
+    .new-tests-line-chart
+      p.text-info ng-if='chart.data.length'
+        strong ng-if='totalCount' {{ totalCount | number }}
+        span ng-if='totalCount === 0' No
+        ng-pluralize count='totalCount' when='{"1": " test in the ", "other": " tests in the "}'
+        span ng-if='project' project.
+        span ng-if='!project' organization.
+      canvas.chart.chart-line height='100' ng-if='chart.data.length' data='chart.data' labels='chart.labels' options='chart.options'
+      form.form role='form' ng-if='chart.data.length'
+        .col-sm-6.col-xs-12 ng-if='!project'
+          project-select prefix='tests-by-week' project='project' organization='organization' model-object='chart.params'
+        .col-xs-12 ng-class='{"col-sm-6": !project }'
+          user-select prefix='tests-by-week' organization='organization' model-object='chart.params'
+      .empty ng-if='chart.data.length === 0'
```

Comment:
The way it's implemented now, this "No recent activity" element is useless. It will never be shown as there will always be data in the array, even if it indicates that there were no tests. I would simply remove it for now as the chart already shows the necessary information.